Query from deep storage doc fixes. (#15382)
apache/druid: docs/querying/query-from-deep-storage.md

Generally, the request body fields are the same between the `sql` and `sql/statements` endpoints.

> Apart from the context parameters mentioned [here](../multi-stage-query/reference.md#context-parameters) there are additional context parameters for `sql/statements` specifically: 

Reviewer: ```suggestion
Apart from the context parameters mentioned [here](../multi-stage-query/reference.md#context-parameters) there are additional context parameters for `sql/statements`: 
```

Use the following endpoint to retrieve results:

```
GET https://ROUTER:8888/druid/v2/sql/statements/QUERYID/results?page=PAGENUMBER&resultFormat=FORMAT
```

Results are returned in JSON format.

> You can use the optional `page`, `resultFormat` parameters to refine your results. 

Reviewer: `resultFormat` doesn't refine the results I think
```suggestion
You can use the optional `page` parameter to refine your results, and `resultFormat` parameter to define the format in which the results will be presented. 
```

* You can retrieve the `page` information for your results by fetching the status of the completed query.

> * For `resultFormat` the following options are supported `arrayLines`,`objectLines`,`array`,`object`,`csv`. Default value is `object`. More documentation present [here](../api-reference/sql-api.md#request-body). 

Reviewer: ```suggestion
* For `resultFormat` the following options are supported `arrayLines`,`objectLines`,`array`,`object`, and `csv`. Default value is `object`. More documentation present [here](../api-reference/sql-api.md#request-body). 
```
